src/D2686.tex

```diff
@@ -382,7 +383,7 @@ \section{Wording for constexpr structured binding}
 \placeholder{S} consist of the \added{\tcode{constexpr} and }\grammarterm{storage-class-specifier}{s} of
 the \grammarterm{decl-specifier-seq} (if any).
 A \cv{} that includes \tcode{volatile} is deprecated;
-see~\ref{depr.volatile.type}.
+see~\ref{depr.volatile.type}. % lah: this cross-ref isn't functioning properly. 
```

Comment:
Yea, this is copy pasted straight from the standard, as long as the committee has enough context to understand where the changes are inserted, it's not an issue - I'll double check it renders properly though